src/jobs/index.html

```diff
@@ -61,201 +61,55 @@
 
 
 <article class="container-md">
-
-  <h1>Join Our Team</h1>
-
-  <p>MyCrypto is one of the world’s leading interfaces for the Ethereum blockchain, offering a suite of tools helpful for any user - from someone who is completely new to the space to the most tenured of crypto enthusiasts. We’re looking to advance our offering to other blockchains and other platforms, all while developing new solutions to meet people’s needs.</p>
-  <h5>Our Products</h5>
-  <ul>
+    <h1>Join Our Team</h1>
+
+    <p>MyCrypto is one of the world’s leading interfaces for the Ethereum blockchain, offering a suite of products for all types of people. We strive to build things that <em>actually</em> solve real problems and iterate quickly. We take an holistic approach to improving this ecosystem and people's lives, prioritizing security, customer support, and being part of this amazing community along with building amazing products.</p>
+    
+    <p>In order to accomplish this, we must have a team compromised of unique, talented individuals. We value different backgrounds and experiences and believe everyone can add value to all aspects of our company. We encourage everyone to grow both personally and professionally—there are no limits to what you can achieve at MyCrypto.</p>
+    
+    <p>Lastly, while we like to move fast, we have processes to make everyone's life easier and use tools to make remote life better. This is a start-up, but you always get paid on time and you aren't living in a lonely bubble. We use Slack for communication, Tandem for hanging out, Clubhouse for project management, Front for support, Confluence for internal documentation, and Github for code and public documentation.</p>
```

Comment:
"This is a start-up, but you always get paid on time and you aren't living in a lonely"
change to
"This is a start-up and a remote environment, but you aren't living in a lonely"

The "paid on time" part goes without saying and also isn't relevant in the context of the rest of the section imo